index.bs

```diff
@@ -316,57 +316,61 @@ Interface {#interface}
 =================
 
 <pre class="idl">
-dictionary NavigatorUABrandVersionDict {
-  required DOMString brand;
+dictionary NavigatorUABrandVersion {
+  DOMString brand;
   DOMString version;
 };
 
+dictionary UADataOptions {
+  boolean platform = false;
+  boolean platformVersion = false;
+  boolean architecture = false;
+  boolean model = false;
+};
+
+dictionary UADataValues {
+  DOMString platform = "";
+  DOMString platformVersion = "";
+  DOMString architecture = "";
+  DOMString model = "";
+};
+
 [Exposed=Window]
 interface NavigatorUAData {
-  readonly attribute FrozenArray&lt;NavigatorUABrandVersionDict&gt; brand;
+  readonly attribute FrozenArray&lt;NavigatorUABrandVersion&gt; UASet;
```

Comment:
Should be `uaSet`; attributes start with lowercase.

Separately this seems pretty confusing. Is "ua" now meant to mean "brand + version"? So "ua" is distinct from "ua data" which contains a set of UAs plus a mobile boolean?

Reply:
Renamed to uaSet. Agree that the naming can be clearer. I'll open a separate issue to discuss it.